docs/use-cases.md

```diff
@@ -1,53 +1,15 @@
-# Use Cases
+# Why Ark?
 
-This doc provides sample Ark commands for the following common scenarios:
-* [Disaster recovery][0]
-* [Cluster migration][1]
+Kubernetes workloads are ephemeral. You describe your cluster's state, and Kubernetes makes it so. So why do you still need to worry about backups and restores? Ark addresses these use cases:
```

Comment:
I wouldn't characterize Kubernetes workloads as ephemeral. I would highlight that Kubernetes lets you define your workloads in a declaratively, which means you describe your cluster's state....